code/mob/living/life/breath.dm

```diff
@@ -102,12 +102,10 @@
 			return // no breathing inside possessed objects
 		else if (istype(owner.loc, /obj/machinery/atmospherics/unary/cryo_cell))
 			return
-
-		// kludge: fake fluid to trigger underwater suffocation check
 		else if (istype(owner.loc, /obj/machinery/bathtub) && owner.lying)
 			var/obj/machinery/bathtub/B = owner.loc
 			if (B.reagents.total_volume > B.suffocation_volume)
-				var/obj/fluid/F = new
+				var/obj/fluid/F = new // used for underwater breathing check
 				F.reagents = owner.loc.reagents
 				underwater = F
```

Comment:
is any of this needed, or could it all be handled in handle_internal_lifeform?

Reply:
The problem I am running into with trying to remove this is the interaction with underwater checks in [breath.dm:131](https://github.com/goonstation/goonstation/blob/fc36eb199cb420764be01e82c82da5f2182eac68/code/mob/living/life/breath.dm#L131=). That code handles returning early when someone is an aquatic mutrantrace, has the 'aquabreath' status (no datum?), or messaging the user if they're drowning.

I'm not sure what the best options are here; rework object handling for `breath.dm` (in a separate PR?), shelve this feature, or go forward with the hacky solution.